### DISCUSSION TITLE
Can Archivebox save pdfs

### DISCUSSION THREAD

**Discussion by TyberiusPrime** [ArchiveBox/ArchiveBox#1204] [General]
I have SAVE_PDF and SAVE_MEDIA set,
but for some reason it's not scrapping a PDF (e.g. https://armyoe.files.wordpress.com/2018/03/1990-fm-22-100.pdf ).

Possibly I'm misunderstanding what it can and can't do?

**Comment by pirate** [comment]
It should be found in the wget output folder. Since that URL is a direct link to an asset (image, video, pdf, executable, etc.), only wget needs to download it. look inside `./archive/<timestamp>/armyoe.files.wordpress.com/2018/03/1990-fm-22-100.pdf`

SAVE_PDF is only for pages that are not *already* PDFs, if converts the HTML to a PDF.
